makefile

```diff
@@ -1,6 +1,7 @@
 # args
 CC = g++
 CFLAGS  = -g -Wall -Weffc++ -std=c++11
+DBGFLAGS = -g -std=c++11
```

Comment:
Never used?

Reply:
I use it to Debug, its more convenient. in the future we might add more, change it, add to it or delete it